substreams/ethereum-rocketpool/src/modules/2_map_protocol_changes.rs

```diff
@@ -122,6 +124,55 @@ fn update_liquidity(
     }
 }
 
+/// Updates rETH contract liquidity based on rETH events.
+/// This is part of the overall Rocket Pool ETH liquidity tracking.
+///
+/// Listens for EtherDeposited and TokensBurned events from the RocketTokenRETH contract and
+/// fetches the updated native ETH balance from the transaction's balance changes.
+///
+/// The reason we do not use the event parameters directly is that they only contain the delta
+/// change, and would force us to start indexing from the very beginning of the protocol.
```

Comment:
Token creation does not necessarily correspond to the beginning of the protocol.
```suggestion
/// change, and would force us to start indexing from token creation
```